From e493a26d005264afe6e4f991a8b6d4d229fbbdd9 Mon Sep 17 00:00:00 2001
From: Sergey Matyukevich <geomatsi@gmail.com>
Date: Thu, 12 Nov 2020 22:36:26 +0300
Subject: [PATCH 2/3] arm: dts: imx6sx: add txdb mailbox

Define all mailboxes in messaging unit dts knob
including optional tx doorbell channel.

Signed-off-by: Sergey Matyukevich <geomatsi@gmail.com>
---
 arch/arm/boot/dts/imx6sx.dtsi | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index 91921e29fa7f..b40404274f4a 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -1626,9 +1626,10 @@
 		 * - general interrupt1: remote proc finish re-init rpmsg stack
 		 *   when A core is partition reset.
 		 */
-		mbox-names = "tx", "rx", "rxdb";
+		mbox-names = "tx", "rx", "txbd", "rxdb";
 		mboxes = <&mu 0 1
 			  &mu 1 1
+			  &mu 2 1
 			  &mu 3 1>;
 		status = "disabled";
 	};
-- 
2.29.2

